.github/workflows/test.yml

```diff
@@ -1,42 +1,42 @@
-# name: Test
+name: Test
 
-# on: pull_request
+on: pull_request
 
-# jobs:
-#   test:
-#     name: Run linters and tests
-#     runs-on: ubuntu-latest
-#     steps:
-#       - name: Cancel Previous Runs
-#         uses: styfle/cancel-workflow-action@0.10.0
-#         with:
-#           access_token: ${{ github.token }}
+jobs:
+  test:
+    name: Run linters and tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
 
-#       - name: Checkout
-#         uses: actions/checkout@v3
+      - name: Checkout
+        uses: actions/checkout@v3
 
-#       - name: Setup node and restore cached dependencies
-#         uses: actions/setup-node@v3
-#         with:
-#           node-version: '16'
-#           cache: 'npm'
+      - name: Setup node and restore cached dependencies
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          cache: 'npm'
 
-#       - name: Install dependencies
-#         run: npm ci
+      - name: Install dependencies
+        run: npm ci
 
-#       - name: Run linter
-#         run: npm run codebase:lint
+      - name: Run linter
+        run: npm run codebase:lint
 
-#       - name: Run unit tests
-#         run: npm test -- --coverage
+      - name: Run unit tests
+        run: npm test -- --coverage
```

Comment:
isn't it?
```suggestion
        run: npm test --coverage
```

Reply:
@nvminhtue it seems we have to add an `--` to add extra parameters to the script of npm. Otherwise it won't work 🙏